react-components/src/architecture/base/commands/CustomBaseInputCommand.ts

```diff
@@ -1,13 +1,23 @@
 /*!
  * Copyright 2024 Cognite AS
  */
+import { type ReactNode } from 'react';
 import { type TranslationInput } from '../utilities/TranslateInput';
 import { RenderTargetCommand } from './RenderTargetCommand';
 
-export type FieldContent = {
-  type: 'text' | 'commentWithButtons';
-  content: string;
-};
+export type FieldContent =
+  | {
+      type: 'text' | 'commentWithButtons' | 'comment';
+      content: string;
+    }
+  | {
+      type: 'customInput';
+      content: ReactNode;
```

Comment:
Nils Petter is not a fan of his architecture being reliant on React - but let's fix that after release

Reply:
yep